src/detect/ScanI2CTwoWire.cpp

```diff
@@ -406,6 +406,7 @@ void ScanI2CTwoWire::scanPort(I2CPort port, uint8_t *address, uint8_t asize)
                 SCAN_SIMPLE_CASE(FT6336U_ADDR, FT6336U, "FT6336U touchscreen found\n");
                 SCAN_SIMPLE_CASE(MAX1704X_ADDR, MAX17048, "MAX17048 lipo fuel gauge found\n");
                 SCAN_SIMPLE_CASE(MLX90614_ADDR, MLX90614, "MLX90614 IR temp sensor found\n");
+                SCAN_SIMPLE_CASE(TPS65233_ADDR, TPS65233, "TPS65233 BIAS-T found\n");
```

Comment:
since TPS65233_ADDR is 0x60 , don't you need to de-conflict with the BME-280 ?

Reply:
Also BME280 has Addr 0x76 or 0x77 - it's just the subtype register that's 0x60. Guarded anyway

Reply:
ah! sorry for making you go to the extra effort :(